### PR TITLE
Saving fix and improvements to sceNp2 and cellCamera

### PR DESCRIPTION
**Pull request by tambry**
:tea:

### CODE REVIEW

**Comment by danilaml** [comment]
Failed to complete on Clang due to multiple defs. Possible due to include of `sceNp.cpp` in np2. Maybe `#pragma once` will fix this.

**Comment by Zangetsu38** [comment]
please implemete this for arkedo 2 :) 
>E {PPU[0x130] Thread (SNKGetMyScore)[0x0005d288]} sceNp: sceNpScoreCreateTitleCtx
E {PPU[0x130] Thread (SNKGetMyScore)[0x0005d1e8]} sceNp: sceNpScoreCreateTransactionCtx
E {PPU[0x130] Thread (SNKGetMyScore)[0x0005d128]} sceNp: sceNpScoreGetRankingByNpId
E {PPU[0x130] Thread (SNKGetMyScore)[0x0005d2c8]} sceNp: sceNpScoreDestroyTransactionCtx
E {PPU[0x130] Thread (SNKGetMyScore)[0x0005d168]} sceNp: sceNpScoreDestroyTitleCtx
E {PPU[0x131] Thread (SNKGetScoreRange)[0x0005d288]} sceNp: sceNpScoreCreateTitleCtx
E {PPU[0x131] Thread (SNKGetScoreRange)[0x0005d1e8]} sceNp: sceNpScoreCreateTransactionCtx
E {PPU[0x131] Thread (SNKGetScoreRange)[0x0005d328]} sceNp: sceNpScoreGetRankingByRange
E {PPU[0x131] Thread (SNKGetScoreRange)[0x0005d2c8]} sceNp: sceNpScoreDestroyTransactionCtx
E {PPU[0x131] Thread (SNKGetScoreRange)[0x0005d168]} sceNp: sceNpScoreDestroyTitleCtx

**Comment by tambry** [comment]
@Zangetsu38 I don't think I'll be implementing those anytime soon, maybe context creation, but the real thing will require quite a bit of development in a couple other areas before getting rankings will be possible.

**Comment by Zangetsu38** [comment]
Ok thank you understand :)

File: rpcs3/Emu/SysCalls/Modules/sceNp2.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
+#include "stdafx.h"
+#include "Emu/Memory/Memory.h"
+#include "Emu/SysCalls/Modules.h"
+
+#include "sceNp.cpp"
```
Comment (Nekotekina) [inline rpcs3/Emu/SysCalls/Modules/sceNp2.cpp]: No, BULLSHIT is here. Who the fuck includes one CPP file into another?

**Pull request merged**